### PR TITLE
sorted translation keys and run all react strings through i18n

### PR DESCRIPTION
**Pull request by eddiekeller**
*Issue #, if available:*

n/a

*Description of changes:*

Sorted all translation strings alphabetically for better readability.
Wrapped all React strings in `I18n` translator.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: packages/core/src/i18n/translations.ts [hunk 1]
```diff
@@ -13,164 +13,163 @@
 
 export const dict = {
   de: {
-    'Sign In': 'Anmelden',
-    'Sign Up': 'Registrieren',
-    'Sign Out': 'Abmelden',
-    'Sign in to your account': 'Melden Sie sich mit Ihrem Account an',
-    Username: 'Benutzername',
```
Comment (ericclemmons) [inline packages/core/src/i18n/translations.ts:20]: How'd you do this?  Did we want to leverage https://www.npmjs.com/package/prettier-plugin-sort-json to do this automatically?

File: packages/core/src/i18n/translations.ts [hunk 2]
```diff
@@ -13,164 +13,163 @@
...
+    'Phone Number': 'Número de teléfono',
+    'Resend a Code': 'Reenviar un código',
+    'Resend Code': 'Mandar codigo otra vez',
+    'Send Code': 'Enviar código',
+    'Sign In': 'Iniciar sesíon',
+    'Sign in to your account': 'Iniciar sesíon',
+    'Sign Out': 'Desconectar',
+    'Sign Up': 'Regístrase',
+    Skip: 'Omitir',
+    Submit: 'Enviar',
```
Comment (ericclemmons) [inline packages/core/src/i18n/translations.ts:166]: What do you think about `consistent` for prettier?
https://prettier.io/docs/en/options.html#quote-props

**Review by ericclemmons — APPROVED** [review]
This looks good @eddiekeller !

I had some questions about making these stylistic improvements automated. What do you think?

File: packages/core/src/i18n/translations.ts (hunk 1 above)
Comment (eddiekeller) [reply packages/core/src/i18n/translations.ts:20]: I just used a VS code extension. Sort JS Objects or something. Super simple.

File: packages/core/src/i18n/translations.ts (hunk 2 above)
Comment (eddiekeller) [reply packages/core/src/i18n/translations.ts:166]: Yeah, I like “consistent.” Let’s update our prettier to use that. I’ll make that change in a coming PR. Good suggestion!

**Pull request merged**